dockerfiles/bloodhound.Dockerfile

```diff
@@ -70,31 +70,31 @@ RUN mkdir -p /opt/bloodhound /etc/bloodhound /var/log
 RUN apk --no-cache add p7zip
 
 # Package Sharphound
-RUN wget https://github.com/SpecterOps/SharpHound/releases/download/$SHARPHOUND_VERSION/SharpHound-$SHARPHOUND_VERSION.zip -O sharphound-$SHARPHOUND_VERSION.zip
-RUN sha256sum sharphound-$SHARPHOUND_VERSION.zip > sharphound-$SHARPHOUND_VERSION.zip.sha256
+RUN wget https://github.com/SpecterOps/SharpHound/releases/download/$SHARPHOUND_VERSION/SharpHound_$SHARPHOUND_VERSION_windows_x86.zip -O sharphound-$SHARPHOUND_VERSION.zip
```

Comment:
I think we need to change this to `${SHARPHOUND_VERSION}` to get CI to pass. It's probably too ambiguous now, and tbh I'm not sure why we didn't do this earlier. Let's make sure there aren't other usages that should be similarly guarded